src/renderer/components/flashing/FlashBuildStatus.tsx

```diff
@@ -0,0 +1,62 @@
+import { InfoCircleOutlined } from "@ant-design/icons";
+import React, { useEffect, useState } from "react";
+import { JobStatus } from "shared/backend";
```

Comment:
You shouldn't import anything from backend

Reply:
This is from shared 😭

Reply:
It's a bit confusing, but backend is only in shared because it's used by both the electric app and the webworker. Maybe it doesn't matter so much but we should probably create some shared types which are not in backend which need to be used by the frontend.